js/foundation.sticky.js

```diff
@@ -46,7 +46,34 @@ class Sticky {
         $('#' + _this.options.anchor).attr({ 'data-mutate': id });
     }
 
+    // Needs refactoring
+    function translateFix() {
+      var topOffset = $(window).scrollTop();
+
+      _this.$element.css('transform','translateY(' + topOffset + 'px)');
+    }
+
```

Comment:
Had to add the function above which is a repeat of the `_translateFix()`method I added to sticky because when I would do `$(window).on('scroll', _this._translateFix);` the `_translateFix` method would not work on scroll. It was giving me an error that said it couldn't run `.css` method on undefined. Obviously something to do with `this` and I tried a few different things but couldn't figure it out.